quantecon/markov/approximation.py

```diff
@@ -299,21 +298,12 @@ def discrete_var(A,
     row_major : bool
         If True then return form the cartesian product state grid using row
         major ordering.  Otherwise use column major.
-    return_sim : bool
-        If True then return the state space simulation generated by the model.
 
     Returns
     -------
-    Pi : array_like(float)
-        A square matrix containing the transition probability
-        matrix of the discretized state.
-    S : array_like(float)
-        An array where element (i,j) of S is the discretized
-        value of the j-th element of x_t in state i. Reducing S to its
-        unique values yields the grid values.
-    Xvec : array_like(float)
-        A matrix of size m x sim_length containing the simulated time
-        series of the m discretized states.
+    mc : MarkovChain
+        An instance of the MarkovChain class that stores the transition
+        matrix and state values returned by the discretization method
```

Comment:
Hi @shlff , can you please put back information about Pi and S?  The user needs to understand what `S[i,j]` represents.